### PR TITLE
Supported threading command line options for other devices

### PR DESCRIPTION
**Pull request by apankratovantonp**
Benchmark application will pass threading command line options to device if it is presented in supported configuration keys list for device

### CODE REVIEW

**Review by ilyachur — APPROVED** [review]
Is this behaviour used only for Hetero/Multi plugins?

**Comment by apankratovantonp** [comment]
> Is this behavior used only for Hetero/Multi plugins?

No. It can be used for any plugin. Even external.

File: inference-engine/samples/benchmark_app/main.cpp [hunk 1]
```diff
@@ -283,6 +283,21 @@ int main(int argc, char *argv[]) {
 
                 if (isFlagSetInCommandLine("nthreads"))
                     device_config[GNA_CONFIG_KEY(LIB_N_THREADS)] = std::to_string(FLAGS_nthreads);
+            } else {
```
Comment (nkogteva) [inline inference-engine/samples/benchmark_app/main.cpp:286]: As far as I understood this code is works for device with names different from `CPU`. I mean for some custom devices which supports CPU settings. Is it correct?

**Review by nkogteva — COMMENTED** [review]
Basically I have no objections. Please update python benchmark_app accordingly.

File: inference-engine/samples/benchmark_app/main.cpp (hunk 1 above)
Comment (apankratovantonp) [reply inference-engine/samples/benchmark_app/main.cpp:286]: Yes, it is correct. Configuration options from the PR are used by IE Streams CPU Executor which can be used by any plugin not only CPU one. So if the device plugin support such options it means that it configure own instance of Streams Executor internally.

**Review by nkogteva — APPROVED** [review]

**Pull request merged**